tests/unittest/_torch/modeling/test_modeling_gemma3.py

```diff
@@ -268,7 +270,9 @@ def test_gemma3_allclose_to_hf(self, scenario: Scenario) -> None:
         model_config = ModelConfig(pretrained_config=gemma3_config,
                                    attn_backend=backend)
         gemma3 = Gemma3ForCausalLM(model_config).to(dtype).to(device)
-        gemma3.load_weights(hf_gemma3.state_dict())
+        weight_mapper = Gemma3HfWeightMapper()
```

Comment:
Can this be done within `load_weigths` to make this an automatic process? At least have the HF as default mapper class. This comment goes for all of the snippets like this in the PR.